### DISCUSSION TITLE
Text on tickets is generated without being bold since last update

### DISCUSSION THREAD

**Discussion by nilskamm** [pretix/pretix#4050] [Q&A] [answered]
Hey everyone! 
We highlighted some texts on our tickets making them "bold" in the Layout-Designer.
When tickets are generated or in the preview these texts are not bold and the rest of the text looks different as well.
As far as i know, we didn´t change anything in the tickets. 
We have this behavior since the last updates.
This problem didn´t occur in 2024.1 and then we updated directly to 2024.3.

This is how it looks in the Layout-Designer:
![grafik](https://github.com/pretix/pretix/assets/75222264/ba9f9607-22db-4202-8bf1-76adc3366829)

And this is the preview:
![grafik](https://github.com/pretix/pretix/assets/75222264/67112c67-b17c-4b11-840b-9bed0559fab8)

I also tested it without our background and the same happens:
![grafik](https://github.com/pretix/pretix/assets/75222264/aa1de9c2-3618-4c71-9ab4-2fff1a424586)

We have pretix 2024.3.0 running with docker like described in the "Small-scale deployment with Docker"-Doc on the pretix website.
Does anyone have an idea what we can do or check? 
Do you need any other information?

Thanks a lot!

**Comment by pc-coholic** [comment, marked as answer]
This was fixed in https://github.com/pretix/pretix/commit/ce826e50f7ba9553c3a4b448d11b10c4b24d784a

You'll have to wait for the next release or update to master... Or if you feel adventurous, you could even cherry-pick that single commit into your install :-)

**Reply by nilskamm** [reply]
Thanks for this insane fast reply! ;-)
I didn´t find this in the issues or pull requests, but good to know, that its already fixed.
I don´t feel adventurous enough at the moment and for us its just cosmetics.
